webdriver-spec.html

```diff
@@ -3919,7 +3919,10 @@
  allow lookup of individual elements and collections of elements.
  Element retrieval searches are performed
  using pre-order traversal of the document’s nodes
- that match the provided selector’s expression.
+ that match the provided selector’s expression, and ignore the value
```

Comment:
Why does this need to be mentioned? Find Elements is not, as currently written, influenced by Page load strategy.